contents/ios/IntroToIos.md

```diff
@@ -0,0 +1,103 @@
+<frontmatter>
+  title: Introduction to iOS App Development
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to iOS Development
+
+Authors: [Bryan Lew](https://github.com/blewjy)
+
+## 1. Overview: What is iOS?
+
+iOS is the mobile operating system that runs on Apple's mobile devices, most notably the iPhone and the iPad. Programs or applications that run on iOS can be downloaded officially from the App Store, and developers of iOS applications can submit their own applications to the App Store to share it with the rest of the world. 
+
+## 2. Why iOS?
+
+In the mobile world, [2 operating systems dominate the whole market share: iOS and Android](https://www.theverge.com/2017/2/16/14634656/android-ios-market-share-blackberry-2016). While Android boasts higher market share in terms of the total number of hardware devices running the operating system, iOS is no slouch in other aspects, especially those important to us sofware engineers.
+
+#### Better compatability and standardisation
+
+iOS applications are specific to only the Apple devices of the iPhone and iPad line, whereas Android runs across numerous different types of devices, including phones, tablets, watches, and more. The large variation of devices poses a compatability problem from a developer's standpoint, because now applications have to be tested and working across all these different devices, and also regularly maintained this way. Each hardware device may possibly be running their own version of Android or will only support up to a particular version, thus Android developers will have to ensure that their application is also compatible across different versions of Android.
+
+With lesser devices to target, iOS developers have more control over their application and are able to customise the application to a greater extent, given that there are less variations (in terms of device compatibility) to worry about. OS versioning in iOS is also more structured, with better backwards-compatability across their devices. Users of iOS devices are always immediately prompted whenever a new version of iOS is available for download, regardless of how old your device is. This means that as developers, we have to worry less about compatability issues, and focus more on the application itself.
+
+#### Lucrative market
+
+While Android has higher market share of devices, iOS applications generate much more revenue than Android, [nearly double](https://techcrunch.com/2018/07/16/apples-app-store-revenue-nearly-double-that-of-google-play-in-first-half-of-2018/). This is largely due to the fact that users of iOS devices are generally more likely to spend on apps, and also because the cost for iOS developers to submit their applicaation to the App Store is higher. 
```

Comment:
generate nearly double the revenue compared to Android

Reply:
> , and also because the cost for iOS developers to submit their applicaation to the App Store is higher. 

application*. Don't get this point because higher cost for developers means less revenue, maybe can elaborate more?